bokehjs/src/lib/models/tiles/tile_renderer.ts

```diff
@@ -52,6 +52,7 @@ export class TileRendererView extends RendererView {
   connect_signals(): void {
     super.connect_signals()
     this.connect(this.model.change, () => this.request_render())
+    this.connect(this.model.tile_source.change, () => this.request_render())
```

Comment:
So, what happens here if the tile source itself is swapped out?

Reply:
Hrm I am not actually sure it is handled currently though I thought it was in the past. I would be OK with merging this and making a follow on issue to investigate more thorough signal disconnect/reconnect (in general, for glyphs and this and anything else)

Reply:
+1

Would be neat if the property could detect a model was being replaced and then dis- and reconnect any signals that were attached to the model being replaced, instead of having to handle this reconnecting logic all over the place. I don't know enough to determine whether that's feasible though.